src/django_aetos/views.py

```diff
@@ -18,12 +19,45 @@ def export_metrics(request):
                 metric["add_help"] = True
             known_metrics.add(metric["name_without_labels"])
             metrics.append(metric)
+    
+    return metrics
+
+def check_auth(request):
+    ip_address = request.META['REMOTE_ADDR']
+    allowed_ips = app_settings.IP_ALLOWLIST
+    
+    if app_settings.ENABLE_IP_ALLOWLIST and ip_address in allowed_ips:
```

Comment:
probably use `is True` here so we don't interpret true-ish values from wrong configs as true.